src/main/java/com/banktaglayouts/BankTagLayoutsPlugin.java

```diff
@@ -6,28 +6,15 @@
 import com.google.common.base.MoreObjects;
 import com.google.common.collect.LinkedListMultimap;
 import com.google.common.collect.Multimap;
-import com.google.common.collect.ObjectArrays;
 import com.google.common.util.concurrent.Runnables;
 import com.google.gson.Gson;
 import com.google.inject.Provides;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.ChatMessageType;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.InventoryID;
-import net.runelite.api.ItemComposition;
-import net.runelite.api.ItemContainer;
-import net.runelite.api.ItemID;
-import net.runelite.api.KeyCode;
-import net.runelite.api.MenuAction;
-import net.runelite.api.MenuEntry;
-import net.runelite.api.MessageNode;
+import net.runelite.api.*;
 import net.runelite.api.Point;
```

Comment:
This doesn't have to be imported if api.* is imported, does it?

Reply:
when I don't import it that way it doesn't compile. It is weird though.

Reply:
ah I see, it's ambiguous with the `java.awt.*` import.